api/src/main/java/org/pmiops/workbench/db/dao/UserService.java

```diff
@@ -0,0 +1,110 @@
+package org.pmiops.workbench.db.dao;
```

Comment:
WorkspaceService follows a interface/impl pattern. Is there a reason to deviate from that pattern here?

Reply:
I primarily care about consistency so things are unambiguous next time someone needs to add a `FooService`. I'd be fine with the latter suggestion (of merging workspace service / impl) if you don't see any value in splitting them. Doesn't necessarily need to be in this PR.

Reply:
OK. I'll merge them in a follow-up.